### ISSUE TITLE
Tiny gravity blocks and slime blocks

### ISSUE THREAD

**Issue by waidesoper** [dannydjdk/Tiny-Redstone#84]
Just to get the full redstone experience, gravity blocks can be used in circuits as a pulse shortener. Slime blocks would allow us to make movable tiny structures.

**Comment by dannydjdk** [comment]
Thanks for the suggestions! Both of these ideas have been in the back of my head for a while. They may be a challenge to implement, so I don't have an eta, but they are definitely on my list of priorities.

**Comment by waidesoper** [comment]
oh, also target blocks have unique properties

**Comment by dannydjdk** [comment]
- [ ] Slime blocks
- [ ] gravity blocks
- [ ] target blocks (maybe)

I'll consider target blocks. It would be a very small target, and since there's no tiny dispenser, it may not be useful for automation. I'm not sure how useful it would be.
Edit: Also, since you can already choose the direction of tiny redstone dust connections, the fact that vanilla target blocks force redstone connections is not relevant in Tiny Redstone.

**Comment by waidesoper** [comment]
Yeah, choosing the direction of Redstone kinda negates target blocks. I forgot about that.

**Comment by Caldoric** [comment]
Don't forget tiny honey blocks to go with the tiny slime blocks!

Speaking of, has anyone tried making tiny flying Redstone machines yet?
